engines/suppliers/Gemfile.lock

```diff
@@ -2,68 +2,93 @@ PATH
   remote: .
   specs:
     suppliers (0.0.1)
-      rails (~> 4.1.7)
+      rails (~> 4.2.5)
```

Comment:
Right, it’s not "used" in the context of the loading application, but there is no way to load a gem by "path" or "git" in .gemspec, so if you need these kind of dependencies for local testing or just to check if all the engines you use are compatible, you are stuck to commit a Gemfile (or else communicate by word to all contributors).


> El 18/1/2016, a las 16:43, Enrico Stano <notifications@github.com> escribió:
> 
> In engines/suppliers/Gemfile.lock <https://github.com/coopdevs/katuma/pull/63#discussion_r50012356>:
> 
> > @@ -2,68 +2,93 @@ PATH
> >    remote: .
> >    specs:
> >      suppliers (0.0.1)
> > -      rails (~> 4.1.7)
> > +      rails (~> 4.2.5)
> nope, Gemfile is only used testing the engine against a dummy application. No use in other cases. In all other cases you use .gemspec file as in any other external gem.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/coopdevs/katuma/pull/63/files#r50012356>.
>